pkgs/development/python-modules/django-currentuser/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, django
+, hatchling
+}:
+let
+  version = "0.6.1";
+in
+buildPythonPackage {
```

Comment:
You could use
```
buildPythonPackage rec {
  pname = "django-currentuser";
  version = "0.6.1";
  format = "pyproject";

  src = fetchFromGitHub {
    [...]
    rev = "v${version}";
```
instead of `let ... in`